doo-swagger.json

```diff
@@ -1912,6 +1916,283 @@
           "type": "mock"
         }
       }
+    },
+    "/webhooks": {
+      "get": {
+        "tags": [
+          "webhooks"
+        ],
+        "description": "Expose a list of configured Webhook Groups for the authenticated user.",
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json",
+          "application/hal+json"
+        ],
+        "parameters": [
+          {
+            "name": "authorization",
+            "in": "header",
+            "description": "Should be `Bearer access_token`",
+            "required": false,
+            "type": "string"
+          },
+          {
+            "name": "Accept",
+            "in": "header",
+            "description": "Default `application/json`",
+            "required": false,
+            "type": "string"
+          },
+          {
+            "name": "Content-Type",
+            "in": "header",
+            "description": "Default `application/json`",
+            "required": false,
+            "type": "string"
+          },
+          {
+            "name": "x-api-key",
+            "in": "header",
+            "description": "API Key",
+            "required": false,
+            "type": "string"
+          },
+          {
+            "name": "page",
+            "in": "query",
+            "description": "Page number",
+            "required": false,
+            "type": "integer"
+          }
+        ],
+        "responses": {
+          "200": {
```

Comment:
It should be specified also all the other response HTTP statuses that BE can respond with (e.g. 400, 401, 500, ...).
It should be done for all the endpoints!
Please check all the endpoints and specify all the response statuses.

`x-amazon-apigateway-integration` section should contain all the same response codes as the endpoint has specified

In case you need to specify response model for standard error/status check `#/definitions/CommonStatusResponse` model and use it if it's satisfied the response

NOTE: option response should have only 200 response code

Reply:
👍